src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1344,8 +1344,10 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
         const auto ptr = args.Pointer();
         const auto point = args.GetCurrentPoint(*this);
+        const auto cursorPosition = point.Position();
+        const auto terminalPosition = _GetTerminalPosition(cursorPosition);

```

Comment:
nit: with these 2 declarations here, the `cursorPosition` and `terminalPos` declarations below (lines 1369 and 1412) can be removed